BUILDIT.md

```diff
@@ -2756,6 +2756,804 @@ Learn more about Ecto with the guides documention, especially the How to section
 https://hexdocs.pm/ecto/getting-started.html (taken from: https://dashbit.co/ebooks/the-little-ecto-cookbook)
 
 
```

Comment:
Table of contents in `BUILDIT.md` not updated to include Edit Timers:
https://github.com/dwyl/mvp/blob/d601dc0863feef976285257883a815acca323258/BUILDIT.md

<img width="1114" alt="image" src="https://user-images.githubusercontent.com/194400/200891409-0bfa8cff-ec22-46c3-b896-e0986c8bbf4a.png">

Definitely don't do this _manually_, use: https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one

Reply:
Fixed by: https://github.com/dwyl/mvp/pull/196/commits/0f0b8139e67e91a77c1f5cee8bf32e04a71dbe22